docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,48 @@
+# Getting started
+
+In this tutorial, we'll walk you through the process of deploying the SAML Integrator charm.
+
+## Requirements
+
+You will need:
+
+* A laptop or desktop running Ubuntu (or you can use a VM).
+* [Juju and Microk8s](https://juju.is/docs/olm/microk8s) installed.
+
+## Deploy this charm
+
+The charm will the deployed in a new model named `saml`:
+
+```
+# Add the model
+juju add-model saml
```

Comment:
I think we should include `-tutorial` or example in the model name here, to make the command later on when you destroy the model less scary/more obvious that it's related to a test deployment.

Reply:
Done